### PR TITLE
feat: added selector tags for e2e testing

### PR DESCRIPTION
**Pull request by JonathanConn**
Added selector tags for e2e testing
Key in the readme

### CODE REVIEW

**Review by 0xAsimetriq — APPROVED** [review]

File: wallets/react-wallet-v2/README.md [hunk 1]
```diff
@@ -48,3 +48,54 @@ Your `.env.local` now contains the following environment variables:
...
+## DOM Tags
+This section is a key of the dom elements used for internal E2E testing
+Accessible with `data-testid`
+
+### Navbar
+| Key | Description |
+| ----------- | ----------- |
+|  `accounts` | Accounts page |
+| `sessions` | Sessions page |
+| `wc-logo` |  WC Connect page |
```
Comment (bkrem) [inline wallets/react-wallet-v2/README.md:61]: Minor: I’d keep this aligned with the underlying functionality like with the other page IDs. `wc-logo` doesn’t convey what that page does.

```suggestion
| `wc-connect` |  WC Connect page |
```

File: wallets/react-wallet-v2/README.md [hunk 2]
```diff
@@ -48,3 +48,54 @@ Your `.env.local` now contains the following environment variables:
 ## Preview of wallet and dapp examples in action
 
 https://user-images.githubusercontent.com/3154053/156764521-3492c232-7a93-47ba-88bd-2cee3f8366d4.mp4
+
+## DOM Tags
+This section is a key of the dom elements used for internal E2E testing
```
Comment (bkrem) [inline wallets/react-wallet-v2/README.md:53]:
```suggestion
This section is a key of the DOM elements used for internal E2E testing
```

File: wallets/react-wallet-v2/src/components/Navigation.tsx [hunk 3]
```diff
@@ -5,19 +5,19 @@ import Link from 'next/link'
...
 
-      <Link href="/sessions" passHref>
+      <Link href="/sessions" passHref data-testid="sessions">
         <a className="navLink">
           <Image alt="sessions icon" src="/icons/sessions-icon.svg" width={27} height={27} />
         </a>
       </Link>
 
-      <Link href="/walletconnect" passHref>
+      <Link href="/walletconnect" passHref data-testid="wc-logo">
```
Comment (bkrem) [inline wallets/react-wallet-v2/src/components/Navigation.tsx:20]: as noted above

```suggestion
      <Link href="/walletconnect" passHref data-testid="wc-connect">
```

**Review by bkrem — APPROVED** [review]
🚀

**Pull request merged**